akka-http2-support/src/main/scala/akka/http/impl/engine/http2/Http2ServerDemux.scala

```diff
@@ -75,10 +75,11 @@ class Http2ServerDemux extends GraphStage[BidiShape[Http2SubStream, FrameEvent,
 
   def createLogic(inheritedAttributes: Attributes): GraphStageLogic =
     new GraphStageLogic(shape) with BufferedOutletSupport { logic ⇒
-      class SubStream(
+      case class SubStream(
```

Comment:
I'm somewhat allergic to keeping `case class` not final unless there's a reason :)